commands/CloudCommands.js

```diff
@@ -125,16 +128,33 @@ CloudCommand.prototype = extend(BaseCommand.prototype, {
 			// todo - move args parsing to a dedicated portion of the command interface (bound to I/O, and separate
 			// from the main command logic.)
 			var idx = utilities.indexOf(args, '--local');
-			this.options.local = idx>=0;
+			if (idx >= 0) {
```

Comment:
it would be good to factor out this repeating pattern of looking for an argument, setting an option and removing the argument if found.

Reply:
Yeh, that makes sense, will take a look at making that change!